addons/website/static/src/scss/website.scss

```diff
@@ -346,7 +346,8 @@ $-seen-urls: ();
 // bootstrap if possible.
 #wrapwrap:not(.o_header_overlay) header, header.o_header_is_scrolled {
     .nav-item > .nav-link > *, .nav-item > .nav-link::after, .js_language_selector span, .badge {
-        color: o-website-value('header-text-color') !important;
+        $-header-text-color: o-website-value('header-text-color');
```

Comment:
Arf big mistake from GDI (and me for merging)... why did we save that *color* value in the website-value map :grimacing: 

Anyways, we'll fix that with the upcoming design system. Could you add a TODO near the `'header-text-color': null,` in primary_variable to indicate it should have been put in the color map and will be in the future?